stock_orderpoint_move_link/models/stock_warehouse_orderpoint.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2019 ForgeFlow S.L.
+#   (http://www.forgeflow.com)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+
+
+class StockWarehouseOrderpoint(models.Model):
+    _inherit = "stock.warehouse.orderpoint"
+
+    def action_view_stock_picking(self):
+        action = self.env.ref("stock.action_picking_tree_all")
+        result = action.read()[0]
+        result["context"] = {}
+        picking_ids = (
+            self.env["stock.move"]
+            .search([("orderpoint_ids", "in", self.id)])
+            .mapped("picking_id")
+        )
+        result["domain"] = "[('id','in',%s)]" % picking_ids.ids
```

Comment:
In order to avoid a search, I would have added the move_ids field on orderpoints with the same table relationship.

That would allows to do:
```suggestion
        result["domain"] = "[('id','in',%s)]" % self.move_ids.mapped("picking_ids").ids
```